csv_detective/detect_fields/FR/geo/region/__init__.py

```diff
@@ -1,8 +1,67 @@
-from frformat import Region
+from frformat import Region, Options
 
 PROPORTION = 1
+ 
+_extra_valid_values_set = {
```

Comment:
Isn't this possible with `frformat` `Options`? How are these values different from the no-accent, no-caps version?